src/lib/components/EventStore/index.js

```diff
@@ -116,6 +116,7 @@ class EventStore extends HTMLElement {
 
     store.setState({
       eventDays: this._days,
+      communityEvents: this._communityEvents,
```

Comment:
Since the community events never change, I'd recommend moving this line to its own `store.setState()` call that happens on line 139